### DISCUSSION TITLE
SpatialExperiment Extensions?

### DISCUSSION THREAD

**Discussion by swbioinf** [csoneson/iSEE#644] [General]
Hi,

Is anyone aware of any work (in development or otherwise) in custom panels to work with SpatialExperiment objects? (Inherits from SummarisedExperiment).

Have done some temorary hacky work to get cosmx data without images into iSEE, and would like to connect with anyone who's working on doing it the 'right' way. 

Thanks

**Comment by kevinrue** [comment]
Dear Sarah,

Thank you very much for opening this discussion. Apologies for the delay in responding, due to family illness including myself (nasty winter flu going around, this side the globe).

Short answer is that yes, you're not alone.
@lgeistlinger @lee-t maybe you want to weigh in.

Longer answer:

I personally haven't found the time to work on it myself yet, but if all goes to plan, I may be funded later this year to work on exactly that.

I also happen to co-organiser EuroBioC2024 in Oxford, UK (https://eurobioc2024.bioconductor.org/), during which I intend to organise a hackathon/discussion around that as well.

**Comment by swbioinf** [comment]
That is great, I hope it goes ahead. Count me in as an alpha tester! :)
